app/assets/stylesheets/welcome.scss

```diff
@@ -1,16 +1,27 @@
+
+
+#box {
+  z-index: 200;
+  height: 300px;
+  margin: 100px 0 -100px 0;
+  text-align: center;
+  width: auto;
+}
+
+
 .vidcontainer {
   position: absolute;
   width: 100%;
   top: 50%;
   left: 50%;
-  margin: 0 0 0 0;
+  margin: -105px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `-105px 0 0` instead of `-105px 0 0 0`